.gitignore

```diff
@@ -1,3 +1,5 @@
 target/
 .idea/
 __pycache__/
+venv
+Cargo.lock
```

Comment:
Cargo.lock should not be added to the `.gitignore`, it pins dependency versions and hashes, and ensures consistent builds.

Reply:
https://doc.rust-lang.org/cargo/faq.html#why-have-cargolock-in-version-control